test/suite/test_checkpoint_snapshot02.py

```diff
@@ -42,17 +42,24 @@ class test_checkpoint_snapshot02(wttest.WiredTigerTestCase):
 
     # Create a table.
     uri = "table:test_checkpoint_snapshot02"
+    backup_dir = "BACKUP"
+    backup_dir2 = "BACKUP2"
 
     format_values = [
         ('column_fix', dict(key_format='r', value_format='8t')),
         ('column', dict(key_format='r', value_format='S')),
         ('integer_row', dict(key_format='i', value_format='S')),
     ]
 
-    scenarios = make_scenarios(format_values)
+    restart_values = [
+        ("crash_restart", dict(restart=True)),
+        ("backup", dict(restart=False)),
+    ]
+
+    scenarios = make_scenarios(format_values, restart_values)
 
     def conn_config(self):
-        config = 'cache_size=10MB,statistics=(all),statistics_log=(json,on_close,wait=1),log=(enabled=true),timing_stress_for_test=[checkpoint_slow]'
+        config = 'cache_size=10MB,statistics=(all),statistics_log=(json,on_close,wait=1),log=(archive=false,enabled=true),timing_stress_for_test=[checkpoint_slow]'
```

Comment:
Why did you add this to turn off archiving? That is not needed for backup.

Reply:
Fixed. Thanks.